components/pages/submission-system/program-submitted-data/SearchBar/FilterModal/index.tsx

```diff
@@ -88,12 +89,29 @@ export default function FilterModal({
       },
     },
   );
+  // console.log('filterDonorIds', filterDonorIds, 'filterSubmitterIds', filterSubmitterIds);
+
+  //number of instances users entered donor id and submitter id that represent the same entry
+  const numOfDoubleCountedId = () => {
+    const doubleCountedResult = searchResultData?.clinicalSearchResults?.searchResults.map(
+      ({ donorId, submitterDonorId }) => ({
+        [donorId]: submitterDonorId,
+      }),
+    ) || [{}];
```

Comment:
```suggestion
    ) || [];
```

I don't think you need the empty object inside the empty array

Reply:
I am getting this error when line 100 is `[]`:
![Screenshot 2023-02-22 at 11 32 17 AM](https://user-images.githubusercontent.com/79996555/220693882-4e04a95f-d428-450b-bc6e-23c54c702503.png)
The error goes away when line 100 is `[{}]`.

Reply:
cool, carry on then 👍